config/routes.rb

```diff
@@ -1,16 +1,24 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-  get :root, to: 'welcome#index'
+  root to: 'welcome#index'
 
-  resources :merchants do
-    resources :items, only: [:index]
-  end
+  get '/merchants', to: 'merchants#index'
+  get '/merchants/new', to: 'merchants#new'
+  post '/merchants', to: 'merchants#create'
+  get '/merchants/:id', to: 'merchants#show', as: :merchant
+  get '/merchants/:id/edit', to: 'merchants#edit'
+  patch '/merchants/:id', to: 'merchants#update'
+  delete '/merchants/:id', to: 'merchants#destroy'
+  get '/merchants/:merchant_id/items', to: 'items#index'
 
-  resources :items, only: [:index, :show] do
-    resources :reviews, only: [:new, :create]
-  end
+  get '/items', to: 'items#index'
+  get '/items/:id', to: 'items#show', as: :item
+  get '/items/:item_id/reviews/new', to: 'reviews#new', as: :new_item_review
+  post '/items/:item_id/reviews', to: 'reviews#create', as: :item_reviews
 
-  resources :reviews, only: [:edit, :update, :destroy]
+  get '/reviews/:id/edit', to: 'reviews#edit', as: :edit_review
+  patch '/reviews/:id', to: 'reviews#update'
+  delete '/reviews/:id', to: 'reviews#destroy', as: :review
 
   get '/cart', to: 'cart#show'
```

Comment:
I attempted to change the cart routes to resources but had trouble because none of the routes match the ones generated from using resources.